doc/guide/install.rst

```diff
@@ -0,0 +1,481 @@
+============
+Installation
+============
+
+The following sections will guide you through the installation of CLIMADA and its dependencies.
+
+.. attention::
+
+    CLIMADA has a complicated set of dependencies that cannot be installed with ``pip`` alone.
+    Please follow the installation instructions carefully!
+    We recommend to use `Anaconda`_ for creating a suitable software environment to execute CLIMADA.
+
+All following instructions should work on any operating system (OS) that is supported by `Anaconda`_, including in particular: **Windows**, **macOS**, and **Linux**.
+
+.. note:: When mentioning the terms "terminal" or "command line" in the following, we are referring to the "Terminal" apps on macOS or Linux and the "Anaconda Prompt" on Windows.
+
+-------------
+Prerequisites
+-------------
+
+* Make sure you are using the **latest version** of your OS. Install any outstanding **updates**.
+* Free up at least 10 GB of **free storage space** on your machine.
+  Anaconda and the CLIMADA dependencies will require around 5 GB of free space, and you will need at least that much additional space for storing the input and output data of CLIMADA.
+* Ensure a **stable internet connection** for the installation procedure.
+  All dependencies will be downloaded from the internet.
+  Do **not** use a metered, mobile connection!
+* Install `Anaconda`_, following the `installation instructions <https://docs.anaconda.com/anaconda/install/>`_ for your OS.
+* Create a **workspace directory**, ideally in subdirectory of your user/home directory.
+  Do **not** use a directory that is synchronized by cloud storage systems like OneDrive, iCloud or Polybox!
+
+.. hint:: If you need help with the vocabulary used on this page, refer to the :ref:`Glossary <install-glossary>`.
+
+.. _install-choice:
+
+Decide on Your Entry Level!
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Depening on your level of expertise, we provide two different approaches:
+
+* If you have never worked with a command line, or if you just want to give CLIMADA a try, follow the :ref:`simple instructions <install-simple>`.
+* If you want to use the very latest development version of CLIMADA or even develop new CLIMADA code, follow the :ref:`advanced instructions <install-advanced>`.
+  If you want to install `CLIMADA Petals`_, also follow these.
+
+Both approaches are not mutually exclusive.
+After successful installation, you may switch your setup at any time.
+
+.. _install-simple:
+
+-------------------
+Simple Instructions
+-------------------
+
+These instructions will install the most recent stable version of CLIMADA without cloning its repository.
+
+#. Download the Anaconda environment specifications for CLIMADA: :download:`env_climada.yml </../requirements/env_climada.yml>`.
```

Comment:
I see. I am a bit hesitant because I do not like instructing people to "mindlessly" copy-paste command line instructions, as they can be quite harmful. The other thing I was thinking about is the availability of `curl`. Can we safely assume that it is available in all setups we encourage? Or do Linux users first have to `apt install curl`?

Reply:
Actually the availability was the argument for using `curl` and not `wget`: `curl` is available on Windows and macOS.
I assume the availability on Linux depends on the distribution. But I guess the number of Linux users that don't have `curl` installed is neglectable. 🧐

Reply:
Then I'll use the `curl` instruction and add to the prerequisites that Linux users have to install `curl`, `python` and `git` 😁